tests/lib/constants.py

```diff
@@ -0,0 +1,164 @@
+"""
+The constants from this file will be useful once CAAS-1623 has been implemented
+also when sandbox environment has been created
+"""
+
+INT_URL = "https://api.service.nhs.uk/caas-published-cohort-definitions"
+CORS_METHODS = "GET, PUT, POST, PATCH, DELETE"
+CORS_MAX_AGE = "3628800"
+CORS_ALLOW_HEADERS = "origin, x-requested-with, accept, " \
+                     "content-type, nhsd-session-urid, " \
+                     "x-correlation-id, authorization"
+CORS_EXPOSE_HEADERS = "x-correlation-id"
+URL_SLUG = "flu-22-to-23"
+TOKENS = [None, "Bearer xyzcba", "Bearer", "junk"]
+METHODS = ["get", "post", "put", "patch", "delete", "head", "options"]
+CORRELATION_IDS = ["76491414-d0cf-4655-ae20-a4d1368472f3"]
+
+
+class Error():
+    def __init__(self, code, status, title, detail):
+        self.code = code
+        self.status = status
+        self.title = title
+        self.detail = detail
+        self.links = {
+            "about": "https://digital.nhs.uk/developer/api-catalogue/published-cohort-definitions"
+        }
+
+
+# invalid value constants
+ERROR_INVALID_VALUE = Error(
```

Comment:
Please check and update all errors. They start with 'CM' which I believe comes from Communications-Manager.

Another aspect is about removing the errors that are not feasible for our API. Cheers.